src/commands/CmdCalendar.cpp

```diff
@@ -544,13 +560,27 @@ std::string CmdCalendar::renderMonths (
         {
           for (auto& hol : Context::getContext ().config)
             if (hol.first.substr (0, 8) == "holiday.")
-              if (hol.first.substr (hol.first.size () - 4) == "date")
+				{
+				  if (hol.first.substr (hol.first.size () - 4) == "date")
               {
-                std::string value = hol.second;
+                auto value = hol.second;
                 Datetime holDate (value.c_str (), Context::getContext ().config.get ("dateformat.holiday"));
                 if (holDate.sameDay (date))
                   cellColor.blend (color_holiday);
               }
+
+              if (hol.first.substr (hol.first.size () - 5) == "start" &&
+                  Context::getContext ().config.has ("holiday." + hol.first.substr (8, hol.first.size () - 14) + ".end"))
+              {
+                auto start = hol.second;
+                auto end = Context::getContext ().config.get ("holiday." + hol.first.substr (8, hol.first.size () - 14) + ".end");
+                Datetime holStart (start.c_str (), Context::getContext ().config.get ("dateformat.holiday"));
+                Datetime holEnd   (end.c_str (), Context::getContext ().config.get ("dateformat.holiday"));
```

Comment:
Perhaps we could move `Context::getContext ().config.get ("dateformat.holiday")` into its own variable? Gets quite repetitive over the code. Would be a nice refactoring win coming for this PR :slightly_smiling_face: